fitbenchmarking/fitbenchmarking
# Consistent spelling

#### Description of Work

Fixes #640 

Makes the package consistent with British Oxford spelling conventions.

#### Testing Instructions

1. Check spelling is consistent

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
